### PR TITLE
Yilinwan 2

### PR DESCRIPTION
**Pull request by TonyW42**
Updated error analysis, classifier, model, and readme (results and CB loss)

### CODE REVIEW

File: experiment/classifier.py [hunk 1]
```diff
@@ -13,9 +13,17 @@
...
 from data import GoEmotionsDataset
-from models import BaselineModel, BaselineEstimator, LabelAwareModel
+from models import BaselineModel, BaselineEstimator
 from utils import make_if_not_exists, seed_everything, config_logging
 
+#added 
+from models import Naive_Label_Model, Naive_Label_Estimator
+from models import var_loss_Model, var_loss_Estimator
+from transformers import BertTokenizer, BertModel
+
```
Comment (yongyi-wu) [inline experiment/classifier.py:23]: We may not want to import them. Just to avoid confusion.

File: experiment/classifier.py [hunk 2]
```diff
@@ -79,12 +87,46 @@ def main():
...
+            lab_id = lab_tok["input_ids"]
+            lab_seg = lab_tok["token_type_ids"]
+            lab_mask = lab_tok["attention_mask"]
+            outputs = mod(lab_id, lab_seg , lab_mask)
+            last_hidden_states = outputs[0].to(device)
+            lab_emb = last_hidden_states[0][1].to(device)
+            label_emb[i] = lab_emb
+    label_emb.to(device)
+    #label_emb.cuda()
+
```
Comment (yongyi-wu) [inline experiment/classifier.py:111]: Did we use this part?

File: experiment/classifier.py [hunk 3]
```diff
@@ -79,12 +87,46 @@ def main():
...
     label_ids = tokenizer.convert_tokens_to_ids(emotions)
+    label_id = torch.tensor(label_ids).type(torch.long).to(device)
+
+    mod.train()
```
Comment (yongyi-wu) [inline experiment/classifier.py:115]: Did we use this part?

File: experiment/classifier.py [hunk 4]
```diff
@@ -111,7 +153,7 @@ def main():
                 min(global_step / warmup_steps, 1 - (global_step - warmup_steps) / train_steps)
             ) # slanted triangular lr
         )
-    estimator = BaselineEstimator( # TODO: change to your estimator!
+    estimator = Naive_Label_Estimator( # TODO: change to your estimator!
```
Comment (yongyi-wu) [inline experiment/classifier.py:156]: Is this the same as BaselineEstimator? Feel free to keep it if it contains new features.

**Review by yongyi-wu — CHANGES_REQUESTED** [review]

**Comment by TonyW42** [comment]
I have fixed these issues. Please see if there is anything else to fix

**Comment by yongyi-wu** [comment]
LGTM
